swaks

```diff
@@ -0,0 +1,3688 @@
+#!/usr/bin/perl
```

Comment:
what is this file? do you use it somewhere?

Reply:
this file was generated when i integrated Mailgun into the app, therefore i think i'll keep it

Reply:
But you cannot add files to your commits and do not know why. I believe this is not needed 3688 lines of code, because you are using simple smtp connection to mailgun, not any gem, etc.